.github/workflows/cd.yml

```diff
@@ -111,15 +109,19 @@ jobs:
     needs: push
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
```

Comment:
This will fetch the newest v2 version so 2.3.4?

Reply:
I guess so. I just want to avoid that it defaults to the master which may cause something like #307 again